nxsdk_modules_contrib/pelenet/lib/anisotropic/connection_asymmetry_landscape.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+#
+# connectivity_assymetry_landscape.py
+#
+# Copyright 2017 Sebastian Spreizer
+# The MIT License
```

Comment:
Please remove the Copyright and License header from the files. Simple sed remove should do.